test/e2e/fixture/zomes/foo/src/lib.rs

```diff
@@ -20,15 +20,25 @@ impl From<&str> for TestString {
 #[hdk_extern]
 fn init(_: ()) -> ExternResult<InitCallbackResult> {
     // grant unrestricted access to accept_cap_claim so other agents can send us claims
-    let mut functions: GrantedFunctions = HashSet::new();
-    functions.insert((zome_info!()?.zome_name, "foo".into()));
-    // functions.insert((zome_info!()?.zome_name, "needs_cap_claim".into()));
+    let mut foo_functions: GrantedFunctions = HashSet::new();
+    foo_functions.insert((zome_info!()?.zome_name, "foo".into()));
     create_cap_grant!(
         CapGrantEntry {
             tag: "".into(),
             // empty access converts to unrestricted
             access: ().into(),
-            functions,
+            functions: foo_functions,
+        }
+    )?;
+
+    let mut emitter_functions: GrantedFunctions = HashSet::new();
+    emitter_functions.insert((zome_info!()?.zome_name, "emitter".into()));
+    create_cap_grant!(
```

Comment:
```suggestion
    // NB: ideally we want to simply create a single CapGrant with both functions exposed,
    // but there is a bug in Holochain which currently prevents this. After this bug is fixed,
    // this can be collapsed to a single CapGrantEntry with two functions.
    // see: https://github.com/holochain/holochain/issues/418
    let mut emitter_functions: GrantedFunctions = HashSet::new();
    emitter_functions.insert((zome_info!()?.zome_name, "emitter".into()));
    create_cap_grant!(
```